src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue31375.cs

```diff
@@ -0,0 +1,24 @@
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue31375 : _IssuesUITest
+	{
+		public Issue31375(TestDevice device) : base(device)
+		{
+		}
+
+		public override string Issue => "[Windows] RefreshView Command executes multiple times when IsRefreshing is set to True";
+
+		[Test]
+		[Category(UITestCategories.RefreshView)]
+		public void VerifyRefreshViewCommandExecution()
```

Comment:
Can include more tests?

Refresh during active refresh:

```
// Start first refresh
App.Tap("RefreshButton");

// While refreshing, trigger again
App.Tap("RefreshButton");
```

Reply:
@jsuarezruiz , 
Based on suggestion, I've modified the changes.